docs/site-components-deployment.md

```diff
@@ -270,14 +275,29 @@ STEAM_PORT=27018
 STV_PORT=27020
 
 RCON_PASSWORD=funny_rcon_password
-SERVER_HOSTNAME=tf2pickup.fi #1
+SERVER_HOSTNAME="tf2pickup.fi #1"
 SERVER_PASSWORD=some_random_password
 STV_NAME=tf2pickup.fi TV
 STV_TITLE=tf2pickup.fi Source TV
-
+# Website API address, must match API_URL from .env file
+# can be set in a server.cfg manually by a variable sm_tf2pickuporg_api_address
 TF2PICKUPORG_API_ADDRESS=https://api.tf2pickup.fi
```

Comment:
Kinda a nit, would be better readability if we had new lines separating the variables 
```env
TF2PICKUPORG_SECRET=yoursuperfunnygameserversecret

# Optional variable, overrides the channel name used for this server, which is derived from the 
# SERVER_HOSTNAME by default (with whitespaces and dots replaced with - etc), in this example
# the default voice channel name would be tf2pickup-fi-1
# can be set in a server.cfg manually by a variable sm_tf2pickuporg_voice_channel_name
TF2PICKUPORG_VOICE_CHANNEL_NAME=

# Optional variable, sets server priority for the server, the default value is 1
# higher value = higher priority
# can be set in a server.cfg manually by a variable sm_tf2pickuporg_priority
TF2PICKUPORG_PRIORITY=1

# Optional variable, used when the tf2pickup.org server is behind a proxy
# the value is an IP address of the game server, for example 177.54.144.126
# can be set in a server.cfg manually by a variable sm_tf2pickuporg_override_internal_address
...
```